backend/spec/agent_spec_helper.rb

```diff
@@ -43,4 +43,11 @@ def add_gender_values
   ge.add_enumeration_value(:value => 'female')
   ge.add_enumeration_value(:value => 'male')
   ge.add_enumeration_value(:value => 'non-binary')
-end
\ No newline at end of file
+end
+
+def add_specific_relator_values
+	spec_rel = Enumeration.find(:name => 'agent_relationship_specific_relator')
```

Comment:
Minor: formatting got a bit weird here. Cleanup?

Reply:
@mark-cooper Got it.